src/main/java/ch/njol/skript/bukkitutil/ItemUtils.java

```diff
@@ -36,27 +36,55 @@ public class ItemUtils {
 
 	/**
 	 * Gets damage/durability of an item, or 0 if it does not have damage.
-	 * @param stack Item.
+	 * @param itemStack Item.
 	 * @return Damage.
 	 */
-	public static int getDamage(ItemStack stack) {
-		ItemMeta meta = stack.getItemMeta();
-		if (meta instanceof Damageable)
+	public static int getDamage(ItemStack itemStack) {
+		ItemMeta meta = itemStack.getItemMeta();
+		if (meta instanceof Damageable) {
 			return ((Damageable) meta).getDamage();
+		}
 		return 0; // Not damageable item
 	}
-	
+
+	/**
+	 * Sets damage/durability of an item if possible.
+	 * @param itemStack Item to modify.
+	 * @param damage New damage. Note that on some Minecraft versions,
+	 * this might be truncated to short.
+	 */
+	public static void setDamage(ItemStack itemStack, int damage) {
+		ItemMeta meta = itemStack.getItemMeta();
+		if (meta instanceof Damageable) {
+			((Damageable) meta).setDamage(damage);
+			itemStack.setItemMeta(meta);
+		}
+	}
+
+	/**
+	 * Gets damage/durability of an item, or 0 if it does not have damage.
+	 * @param itemType Item.
+	 * @return Damage.
+	 */
+	public static int getDamage(ItemType itemType) {
+		ItemMeta meta = itemType.getItemMeta();
+		if (meta instanceof Damageable) {
+			return ((Damageable) meta).getDamage();
+		}
+		return 0; // Not damageable item
+	}
+
 	/**
 	 * Sets damage/durability of an item if possible.
-	 * @param stack Item to modify.
+	 * @param itemType Item to modify.
 	 * @param damage New damage. Note that on some Minecraft versions,
 	 * this might be truncated to short.
 	 */
-	public static void setDamage(ItemStack stack, int damage) {
-		ItemMeta meta = stack.getItemMeta();
+	public static void setDamage(ItemType itemType, int damage) {
```

Comment:
We should keep the old method and deprecate it in case any addon was using it, then we can remove it later.

Reply:
both methods still exist, I believe it's just git fuckin up

Reply:
You changed the parameters though, so it's a different method now.

Reply:
not quite sure what you mean the only thing changed in the methods are the field names `stack -> itemStack`

Reply:
![image](https://github.com/SkriptLang/Skript/assets/10354869/7c6054f3-b6ca-4050-b2df-4c38fa48835e)

Reply:
and as I said git is messing up
![image](https://github.com/SkriptLang/Skript/assets/81952476/77aeaefe-cecd-4371-a5d2-6bd790482291)

Reply:
oh it shoved it all the way up there, i see